internal/peer/common/common.go

```diff
@@ -187,6 +197,15 @@ func SetBCCSPKeystorePath() {
 	}
 }
 
+// SetBCCSPPKCS11Keys sets all of the keys for the PKCS11 BCCSP provider
+func SetBCCSPPKCS11Keys() {
+	keys := []string{"peer.BCCSP.Default", "peer.BCCSP.PKCS11.Hash", "peer.BCCSP.PKCS11.Security", "peer.BCCSP.PKCS11.Library", "peer.BCCSP.PKCS11.Label", "peer.BCCSP.PKCS11.Pin"}
```

Comment:
not enough peer.BCCSP.PKCS11.Altid

and not just his

Reply:
I added support for peer.BCCSP.PKCS11 SoftwareVerify, Immutable, AltID, and KeyIDs. KeyIDs requires the use of specifying a json string for the CORE_PEER_BCCSP_PKCS11 environment variable.